src/refresh/debug_text.c

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright (C) 1997-2001 Id Software, Inc.
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "shared/shared.h"
+#include "shared/debug.h"
+#include "common/cvar.h"
+#include "common/math.h"
+#include "common/prompt.h"
+#include "debug_fonts/cursive.h"
```

Comment:
Do we really need all these fonts just for debug purposes? Maybe just keep one or two?

Reply:
Not strictly. They're fun, though 😅

Reply:
Let's keep 1-3 fonts that are most practical and remove the rest.

Reply:
Well, ok.